package.json

```diff
@@ -4,14 +4,17 @@
   "main": "index.js",
   "scripts": {
     "clean": "rimraf dist",
-    "dev": "ts-node src/index.ts",
+    "dev": "dotenv ts-node src/index.ts",
     "test": "NODE_ENV=test jest",
     "coverage": "yarn codecov",
     "deploy:staging": "now -e APP_ID=@oc-bot-staging-app-id -e WEBHOOK_SECRET=@oc-bot-staging-webhook-secret -e PRIVATE_KEY=@oc-bot-staging-private-key -e SENTRY_DSN=@oc-bot-staging-sentry-dsn --target staging && now alias bot-staging.opencollective.com",
     "deploy:production": "now --target production"
   },
   "dependencies": {
+    "@types/dotenv": "^6.1.1",
```

Comment:
We don't need that if we use `dotenv-cli`.

Reply:
Alright . Guess I forgot to take it out